config/crd-overlay.yml

```diff
@@ -18,3 +18,13 @@ spec:
 #@overlay/remove
 #@overlay/match missing_ok=True
 status:
+
+#@overlay/match by=overlay.subset({"metadata":{"name":"internalpackagemetadata.internal.packaging.carvel.dev"}}), expects="0+"
+---
+metadata:
+  #@overlay/replace
+  name: internalpackagemetadatas.internal.packaging.carvel.dev
+spec:
+  names:
+    #@overlay/replace
```

Comment:
no need for replace since it's a scalar.

Reply:
Thanks, addressed